### PR TITLE
DEV-12881 Adding ability to specify additional logging properties to …

### PR DESCRIPTION
**Pull request by simon-technocloud**
Tweaks to improve logging and reusability

### CODE REVIEW

**Review by neilD2 — APPROVED** [review]
LGTM

**Review by nroper — APPROVED** [review]
LGTM

File: api-services/helpers.js [hunk 1]
```diff
@@ -32,7 +32,7 @@ export function addMaxBodyLengthToRequestConfiguration (requestConfiguration, ma
     return requestConfiguration
 }
 
-export function buildApiRequestConfig (url, headers, data) {
+export function buildApiRequestConfig (url, headers, data, loggingProperties) {
```
Comment (richmilton) [inline api-services/helpers.js:35]: Might be worth having `loggingProperties = null` so devs know that it's an optional param (ie won't break anything if absent)

**Review by richmilton — APPROVED** [review]
LGTM - one minor suggestion

File: api-services/helpers.js (hunk 1 above)
Comment (simon-technocloud) [reply api-services/helpers.js]: Updating to make optionals null

**Pull request merged**